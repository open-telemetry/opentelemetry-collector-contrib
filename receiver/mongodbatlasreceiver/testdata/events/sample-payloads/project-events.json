[
    {
        "alertId": "63975427b7942a211665ae20",
        "alertConfigId": "",
        "created": "2022-12-12T16:29:43Z",
        "eventTypeName": "ALERT_ACKNOWLEDGED_AUDIT",
        "groupId": "5bc762b579358e3332046e6a",
        "hostname": "",
        "id": "639756f71d4cad6afbce4cae",
        "links": [
            {
                "rel": "self",
                "href": "https://cloud.mongodb.com/api/atlas/v1.0/groups/5bc762b579358e3332046e6a/events/639756f71d4cad6afbce4cae"
            }
        ],
        "remoteAddress": "72.44.116.110",
        "userId": "5bc76130ff7a25df17bbd9d9",
        "username": "devops@open-telemetry.org"
    },
    {
        "alertId": "",
        "alertConfigId": "",
        "created": "2022-12-12T16:17:43Z",
        "currentValue": {
            "number": 6,
            "units": "RAW"
        },
        "eventTypeName": "OUTSIDE_METRIC_THRESHOLD",
        "groupId": "5bc762b579358e3332046e6a",
        "hostname": "atlas-yjc7yl-shard-00-02.bzrvwof.mongodb.net",
        "id": "63975427b7942a211665ae1d",
        "links": [
            {
                "rel": "self",
                "href": "https://cloud.mongodb.com/api/atlas/v1.0/groups/5bc762b579358e3332046e6a/events/63975427b7942a211665ae1d"
            }
        ],
        "metricName": "CONNECTIONS",
        "orgId": "5bc76130ff7a25df17bbd9de",
        "Port": 27017,
        "replicaSetName": "atlas-yjc7yl-shard-0"
    },
    {
        "alertId": "",
        "alertConfigId": "6331e126aecbcc4b2c1bb666",
        "created": "2022-12-12T16:17:06Z",
        "eventTypeName": "ALERT_CONFIG_CHANGED_AUDIT",
        "groupId": "5bc762b579358e3332046e6a",
        "hostname": "",
        "id": "63975402294a8a205ec865e3",
        "links": [
            {
                "rel": "self",
                "href": "https://cloud.mongodb.com/api/atlas/v1.0/groups/5bc762b579358e3332046e6a/events/63975402294a8a205ec865e3"
            }
        ],
        "remoteAddress": "72.44.116.110",
        "userId": "5bc76130ff7a25df17bbd9d9",
        "username": "devops@open-telemetry.org"
    }
]
