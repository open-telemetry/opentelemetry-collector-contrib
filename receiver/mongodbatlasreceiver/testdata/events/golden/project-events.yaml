resourceLogs:
  - resource:
      attributes:
        - key: mongodbatlas.project.name
          value:
            stringValue: test-project
        - key: mongodbatlas.org.id
          value:
            stringValue: test-org-id
    scopeLogs:
      - logRecords:
          - attributes:
              - key: event.domain
                value:
                  stringValue: mongodbatlas
              - key: type
                value:
                  stringValue: ALERT_ACKNOWLEDGED_AUDIT
              - key: id
                value:
                  stringValue: 639756f71d4cad6afbce4cae
              - key: group.id
                value:
                  stringValue: 5bc762b579358e3332046e6a
              - key: alert.id
                value:
                  stringValue: 63975427b7942a211665ae20
              - key: user.name
                value:
                  stringValue: devops@open-telemetry.org
              - key: user.id
                value:
                  stringValue: 5bc76130ff7a25df17bbd9d9
              - key: remote.ip
                value:
                  stringValue: 72.44.116.110
            body:
              stringValue: '{"alertId":"63975427b7942a211665ae20","alertConfigId":"","created":"2022-12-12T16:29:43Z","eventTypeName":"ALERT_ACKNOWLEDGED_AUDIT","groupId":"5bc762b579358e3332046e6a","hostname":"","id":"639756f71d4cad6afbce4cae","links":[{"rel":"self","href":"https://cloud.mongodb.com/api/atlas/v1.0/groups/5bc762b579358e3332046e6a/events/639756f71d4cad6afbce4cae"}],"remoteAddress":"72.44.116.110","userId":"5bc76130ff7a25df17bbd9d9","username":"devops@open-telemetry.org"}'
            observedTimeUnixNano: "1680099677582009000"
            spanId: ""
            timeUnixNano: "1670862583000000000"
            traceId: ""
        scope: {}
      - logRecords:
          - attributes:
              - key: event.domain
                value:
                  stringValue: mongodbatlas
              - key: type
                value:
                  stringValue: OUTSIDE_METRIC_THRESHOLD
              - key: id
                value:
                  stringValue: 63975427b7942a211665ae1d
              - key: group.id
                value:
                  stringValue: 5bc762b579358e3332046e6a
              - key: net.peer.name
                value:
                  stringValue: atlas-yjc7yl-shard-00-02.bzrvwof.mongodb.net
              - key: net.peer.port
                value:
                  intValue: "27017"
              - key: metric.name
                value:
                  stringValue: CONNECTIONS
              - key: replica_set.name
                value:
                  stringValue: atlas-yjc7yl-shard-0
              - key: metric.value
                value:
                  doubleValue: 6
              - key: metric.units
                value:
                  stringValue: RAW
            body:
              stringValue: '{"alertId":"","alertConfigId":"","created":"2022-12-12T16:17:43Z","currentValue":{"number":6,"units":"RAW"},"eventTypeName":"OUTSIDE_METRIC_THRESHOLD","groupId":"5bc762b579358e3332046e6a","hostname":"atlas-yjc7yl-shard-00-02.bzrvwof.mongodb.net","id":"63975427b7942a211665ae1d","links":[{"rel":"self","href":"https://cloud.mongodb.com/api/atlas/v1.0/groups/5bc762b579358e3332046e6a/events/63975427b7942a211665ae1d"}],"metricName":"CONNECTIONS","orgId":"5bc76130ff7a25df17bbd9de","Port":27017,"replicaSetName":"atlas-yjc7yl-shard-0"}'
            observedTimeUnixNano: "1680099677582009000"
            spanId: ""
            timeUnixNano: "1670861863000000000"
            traceId: ""
        scope: {}
      - logRecords:
          - attributes:
              - key: event.domain
                value:
                  stringValue: mongodbatlas
              - key: type
                value:
                  stringValue: ALERT_CONFIG_CHANGED_AUDIT
              - key: id
                value:
                  stringValue: 63975402294a8a205ec865e3
              - key: group.id
                value:
                  stringValue: 5bc762b579358e3332046e6a
              - key: alert.config.id
                value:
                  stringValue: 6331e126aecbcc4b2c1bb666
              - key: user.name
                value:
                  stringValue: devops@open-telemetry.org
              - key: user.id
                value:
                  stringValue: 5bc76130ff7a25df17bbd9d9
              - key: remote.ip
                value:
                  stringValue: 72.44.116.110
            body:
              stringValue: '{"alertId":"","alertConfigId":"6331e126aecbcc4b2c1bb666","created":"2022-12-12T16:17:06Z","eventTypeName":"ALERT_CONFIG_CHANGED_AUDIT","groupId":"5bc762b579358e3332046e6a","hostname":"","id":"63975402294a8a205ec865e3","links":[{"rel":"self","href":"https://cloud.mongodb.com/api/atlas/v1.0/groups/5bc762b579358e3332046e6a/events/63975402294a8a205ec865e3"}],"remoteAddress":"72.44.116.110","userId":"5bc76130ff7a25df17bbd9d9","username":"devops@open-telemetry.org"}'
            observedTimeUnixNano: "1680099677582009000"
            spanId: ""
            timeUnixNano: "1670861826000000000"
            traceId: ""
        scope: {}
